master/master.cfg

```diff
@@ -110,6 +110,17 @@ c["configurators"] = [
     )
 ]
 
+c["caches"] = {
```

Comment:
Can you please add a comment explaining how you chose these numbers?

I'm fine with "These are mostly made up numbers just to see if there is any noticeable effect on either performance or memory usage."